setup.py

```diff
@@ -95,14 +128,15 @@ def build_extensions(self):
         opts = self.c_opts.get(ct, [])
         link_opts = self.l_opts.get(ct, [])
         if ct == 'unix':
-            opts.append(cpp_flag(self.compiler))
-            if has_flag(self.compiler, '-fvisibility=hidden'):
-                opts.append('-fvisibility=hidden')
+            if sys.platform != 'darwin':
+                opts.append(cpp_flag(self.compiler))
+            #if has_flag(self.compiler, '-fvisibility=hidden'):
+            #    opts.append('-fvisibility=hidden')
 
         for ext in self.extensions:
             ext.define_macros = [('VERSION_INFO', '"{}"'.format(self.distribution.get_version()))]
-            ext.extra_compile_args = opts
-            ext.extra_link_args = link_opts
+            ext.extra_compile_args += opts
+            ext.extra_link_args += link_opts
```

Comment:
I had issues with "-framework" lines not reaching the linker. I' believe this was related to that?

Reply:
Yes, that seems right. I copied this as-is from the pybind11 docs, probably worth reporting it.

Reply:
Would be nice if *you* could report it; I never worked with pybind11 (except for the hacks in this PR).

Reply:
Looks like they factored that into a component now https://github.com/pybind/python_example/blob/master/setup.py. At some point I'll probably switch to that.